pkg/products/threescale/reconciler.go

```diff
@@ -1444,3 +1457,42 @@ func (r *Reconciler) reconcileSubscription(ctx context.Context, serverClient k8s
+		cl.Spec.ApplicationMenu.Section = "Red Hat Applications"
```

Comment:
I think 3-Scale would be better under "Red Hat Integration" - I'm not 100% sure but it seems it is a part of our integration products https://www.redhat.com/en/products/integration

Reply:
This I am very open to change on and as on that product page its under integration, that's the menu I will move it under. Change coming soon.